app/src/views/features/sessions/SessionsIndexPageContainer/NetworkSessions/SessionSaveModal.tsx

```diff
@@ -21,18 +23,33 @@ interface Props {
 }
 
 const SessionSaveModal: React.FC<Props> = ({ har, isVisible, closeModal, onSave }) => {
+  const navigate = useNavigate();
   const dispatch = useDispatch();
-
   const networkSessionTooltipShown = useSelector(getIsNetworkTooltipShown);
-
   const [name, setName] = useState<string>("");
 
   const handleSaveRecording = useCallback(async () => {
     const id = await saveNetworkSession(name, har);
     if (onSave) {
       onSave(id);
     } else {
-      toast.success("Network session successfully saved!");
+      toast.success({
+        key: "view_network_session",
+        content: (
+          <span>
+            Network session saved successfully.{" "}
```

Comment:
I think we need more space between text and link.
Let's add at least 24px gap and change link label to `View session`.
And, ask for a design review.

<img width="393" alt="image" src="https://github.com/requestly/requestly/assets/6367566/a797e30e-b8fe-4ca7-b8ae-dbeaf8973e18">

You may wrap the text and link inside a <Space gap={24}> or a flex to add the gap.